string satname
time time_start
time time_stop
float32 apogey